s4e-web/src/app/views/map-view/state/overlay/overlay.factory.spec.ts

```diff
@@ -4,5 +4,10 @@ import {Overlay} from './overlay.model';
 export const OverlayFactory = Factory.makeFactory<Overlay>({
   id: Factory.each(i => `overlay:${i}`),
   caption: Factory.each(i => `Overlay #${i}`),
-  type: 'wms'
+  type: 'wms',
```

Comment:
**Keep it all as it is, but for the future purposes**

From now on all overlays will be of `wms` type. There will be `ownerType` instead that will say if it's `personal`, `institutional`, or `global` usage one.

Reply:
Yes, I'm aware - but that will be changed after API is ready